posts/Yoann/2018-07-25-hibernate-search-5-10-3-Final.adoc

```diff
@@ -0,0 +1,43 @@
+= Third maintenance release for Hibernate Search 5.10
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+While we are still working on Hibernate Search 6 to get it ready for an Alpha,
+things are relatively quiet on the Search 5 side.
+However, we did not forget about it: we just published Hibernate Search version 5.10.3.Final,
+the third maintenance release of Hibernate Search 5.10.
+This release upgrades Hibernate Search to Hibernate ORM 5.3.3.Final, and adds a few minor improvements.
+
+== What's new?
+
+* https://hibernate.atlassian.net/browse/HSEARCH-3234[HSEARCH-3234]
+and https://hibernate.atlassian.net/browse/HSEARCH-3210[HSEARCH-3210]:
+Upgrade to ORM 5.3.3.Final and Hibernate Common Annotations 5.0.4.Final.
+Hibernate ORM 5.3.3.Final includes quite a few bug fixes and improvements,
+in particular related to backward compatibility, so be sure to http://hibernate.org/orm/releases/5.3/[check it out]!
```

Comment:
`in particular related to backward compatibility` -> we mostly did this work so that the bytecode transformer of WF could use the reintroduced methods, not sure it makes sense to underline it for the standard user.